backend/danswer/background/update.py

```diff
@@ -60,7 +61,7 @@
 def _should_create_new_indexing(
     cc_pair: ConnectorCredentialPair,
     last_index: IndexAttempt | None,
-    model: EmbeddingModel,
+    model: SearchSettings,
```

Comment:
Should this param still be "model"?

Reply:
Updated to be clearer! Should be clearer now